tests/external_httpx/test_client.py

```diff
@@ -87,3 +98,115 @@ async def test_async_client():
 
     responses = loop.run_until_complete(test_async_client())
     assert all(response.status_code == 200 for response in responses)
+
+
+@pytest.mark.parametrize(
+    "distributed_tracing,span_events",
+    (
+        (True, True),
+        (True, False),
+        (False, False),
+    ),
+)
+def test_sync_cross_process_request(httpx, server, distributed_tracing, span_events):
+    @override_application_settings(
+        {
+            "distributed_tracing.enabled": distributed_tracing,
+            "span_events.enabled": span_events,
+        }
+    )
+    @validate_transaction_errors(errors=[])
+    @background_task()
```

Comment:
Nit: we should use `name=` here to avoid <local> in the naming if we ultimately build assertions on the metrics.